packages/kbn-cli-dev-mode/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@kbn/cli-dev-mode",
+  "main": "./target/index.js",
+  "types": "./target/index.d.ts",
+  "version": "1.0.0",
+  "license": "SSPL-1.0 OR Elastic License 2.0",
+  "private": true,
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc",
+    "kbn:bootstrap": "yarn build"
+  },
+  "kibana": {
+    "devOnly": true
+  },
+  "dependencies": {
+    "@kbn/config": "link:../kbn-config",
+    "@kbn/config-schema": "link:../kbn-config-schema",
+    "@kbn/logging": "link:../kbn-logging",
+    "@kbn/http-tools": "link:../kbn-server-http-tools",
+    "@kbn/optimizer": "link:../kbn-optimizer",
+    "@kbn/std": "link:../kbn-std",
+    "@kbn/dev-utils": "link:../kbn-dev-utils",
+    "@kbn/utils": "link:../kbn-utils"
+  },
+  "devDependencies": {
+    "@kbn/utility-types": "link:../kbn-utility-types"
+  }
```

Comment:
Looks like `@kbn/server-http-tools` didn't get renamed and `@kbn/utility-types` is not used.